variables.tf

```
@@ -32,3 +32,8 @@ variable "tags" {
   default     = {}
   description = "Additional tags (e.g. `map(`BusinessUnit`,`XYZ`)"
 }
+
+variable "convertcase" {
+  description = "Convert tags to lower case"
```

It's not just tags, right? It's everything. 
```suggestion
  description = "Convert fields to lower case"
```